ironfish-rust-nodejs/src/structs/transaction.rs

```diff
@@ -460,6 +460,19 @@ impl NativeUnsignedTransaction {
 
         Ok(Buffer::from(vec))
     }
+
+    #[napi]
+    pub fn add_signature(&mut self, signatures: String) -> Result<Buffer> {
```

Comment:
Why are you using a `String` here? I think you should be using a `JsBuffer` to avoid ser/de. Not sure why we use string in some places and not others:
ironfish-rust-nodejs/src/structs/transaction.rs

Unless of course the data is coming from ledger as a string, then it doesn't matter if you do serde at this layer.

Reply:
Data from the ledger is coming in as a buffer, so I'll change it to JsBuffer